### ISSUE TITLE
Add support for multi-word string imports in lists & sets 

### ISSUE THREAD

**Issue by daveharig** [danishi/dynamodb-csv#216]
Lists and sets delineate items with a space which limits the functionality to single word strings.  

Would be valuable to support multi-word string imports by delineating with commas or quotes.

**Comment by danishi** [comment]
@daveharig 
Absolutely!

I plan to make changes so that the delimiter can be defined in the CSV_SPEC.
Please wait a moment.

**Comment by daveharig** [comment]
Sounds great. For now I did this...

```
    elif spec == "SL":  # StringList
        return row[key].split('|')
    elif spec == "SS":  # StringSet
        return set(row[key].split('|'))
        ```

**Comment by danishi** [comment]
@daveharig 
Thank you for waiting!

I resolved this issue in  [version 1.5.4](https://github.com/danishi/dynamodb-csv/releases/tag/v1.5.4).
It is already possible to install from PyPI.

Just add this to the spac file you are using.

```ini
[DELIMITER_OPTION]
DelimiterCharacter=|
```

I hope this helps you.
And thank you so much for being my first GitHub sponsor💖
